bigbluebutton-html5/private/config/public/app.yaml

```diff
@@ -25,3 +25,30 @@ app:
   basename: '/html5client'
 
   defaultLocale: 'en'
+  #default settings for session storage
```

Comment:
For now it is just the default values for the "Setting object" and I don't think some of them belong here, but because of a lack of better place to have them, I decided to have them here, for instance `inputDeviceId`, `outpudDeviceId`. 
I am not sure of having those on Settings, they don't seem like settings to my, for me they are more like application parameters that are not changed if the user desire, while settings sound like something you can change somehow.